tests/unit/pii/test_scrubber.py

```
@@ -229,6 +229,38 @@ def test_snapshot_to_container_noop_without_registration(self):
+        """Verify that a singleton scrubber does not leak PII across requests."""
+        import contextvars
+
+        s = _scrubber(_make_rule("email", "scrub"))
+
+        container_a: dict[str, str] = {}
+        container_b: dict[str, str] = {}
+
+        ctx_a = contextvars.copy_context()
+        ctx_b = contextvars.copy_context()
+
+        def request_a():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `request_a`

Add return type annotation: `None`

(ANN202)